basics_dev/coding-style.md

```diff
@@ -165,7 +165,7 @@ Security coding guidelines
        height = untrusted_conf.height;
     ~~~
 
--   Use another variables, without the `untrusted_` prefix to hold the sanitized values, as shown above.
```

Comment:
Okay, the sentence is wrong due to mismatch in plurality, but I think change is in the wrong direction.

`untrusted_foo` and `foo` are most definitely *not* equivalent from a security perspective, and accidentally interchanging them might be fatal.

Perhaps "Use other variables," or "Use another variable," or "Use corresponding variables,"?